resolve/index.go

```diff
@@ -91,18 +105,10 @@ type ruleRecord struct {
 	// Used to build a map from ImportSpecs to ruleRecords.
 	importedAs []ImportSpec
 
-	// embeds is the transitive closure of labels for rules that this rule embeds
-	// (as determined by the Embeds method). This only includes rules in the same
-	// language (i.e., it includes a go_library embedding a go_proto_library, but
-	// not a go_proto_library embedding a proto_library).
+	// Embeds is the set of labels (of any language) that this rule directly Embeds,
```

Comment:
Is it true that this covers all languages? The logic below looks this would only contain labels for the same language.

Reply:
This field is whatever `Resolver.Embeds` returns (normalized a bit so the labels are always absolute).

We really have no idea what the label is, what `ImportSpec`s that label will produce or what language those `ImportSpec`s will be.

I find a lot of this terminology confusing so let me know if you can think of better phrasing for any of these comments while we're editing them.